.github/workflows/ci.yml

```diff
@@ -33,7 +30,7 @@ jobs:
       - name: Synthesize CDK app
         run: npx cdk synth
 
-      # - name: Archive cdk.out
```

Comment:
no point in doing this, I found out, if the size of the cdk.out folder is greater than some 10 MB (which is always the case).